cluster/gce/config-default.sh

```diff
@@ -194,6 +194,24 @@ if [[ ${ENABLE_METADATA_CONCEALMENT:-} == "true" ]]; then
   PROVIDER_VARS="${PROVIDER_VARS:-} ENABLE_METADATA_CONCEALMENT METADATA_CONCEALMENT_NO_FIREWALL"
 fi
 
+# Enable AESGCM encryption of secrets by default.
+ENCRYPTION_PROVIDER_CONFIG="${ENCRYPTION_PROVIDER_CONFIG:-}"
+if [[ -z "${ENCRYPTION_PROVIDER_CONFIG}" ]]; then
+    ENCRYPTION_PROVIDER_CONFIG=$(cat << EOM | base64 -w 0
```

Comment:
Have you tested this on a mac? See https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/util.sh#L1026 where I had to work around apple's base64 with `tr`.

Reply:
Thanks for catching this; I was not aware of -w not being available on Macs.
Replace -w 0 with | tr -d '\r\n'